build/config/phone/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "dom.presentation.discovery.enabled": true
```

Comment:
We may not need this entry if we don't observe it. BTW, will this value be able to change in the settings app?